jax/dtypes.py

```diff
@@ -26,4 +26,6 @@
     prng_key as prng_key,
     result_type as result_type,
     scalar_type_of as scalar_type_of,
+    float8_e4m3fnuz as float8_e4m3fnuz,
+    float8_e5m2fnuz as float8_e5m2fnuz,
```

Comment:
Let's avoid exporting these here (and if we do choose to export these, we should export all float8 dtypes)